jumpscale/packages/tfgrid_solutions/chats/cryptpad_deploy.py

```diff
@@ -49,7 +50,7 @@ def cryptpad_name(self):
                     break
                 valid = True
 
-    @chatflow_step(title="Container resources")
+    @chatflow_step(title="")
```

Comment:
it's not a step actually, we can merge it with `volume_details` method and give it a title `Solution Size` or so